posts/2024-03-15-jfokus-2024.adoc

```diff
@@ -0,0 +1,66 @@
+= JFokus 2024
+jdijt
+v1.0, 2024-03-13
+:title: JFokus 2024
+:imagesdir: ../media/2024-03-13-jfokus-2024
+:lang: en
+:tags: [conference, stockholm, sweden, java, jfokus]
+
+Recently myself and two other Lunatech colleagues had the opportunity to visit JFokus.
+JFokus is a conference that is held annually in the first week of February in Stockholm, Sweden. 
+
+In this post I will discuss my general experience of the conference, 
+followed by short discussions of two talks that I attended.
+
+== The Conference
+JFokus is held at the Waterfront Conference Center in Stockholm, located almost on top of the Stockholm Central Station.
+Catering wise the quality is excellent and the supply of coffee appears to be infinite which is always a good thing.
+Overall attending the conference was a very smooth and comfortable experience.
+I would like to compliment the organizers and volunteers on the quality of the event!
+
+Content wise a broad range of subjects is covered with a focus on java, the JVM, and the surrounding ecosystem.
+This year there were also a good number of talks discussing AI, both how to use it within your projects and how to use to it in your own coding process.
+With over 70 talks scheduled attending all the talks that look interesting is an impossible assignment!
+Luckily all the talks are recorded and published on YouTube. 
```

Comment:
```suggestion
Luckily all the talks were recorded and are published on YouTube.
```

Perhaps reference the link to the youtube videos you've provided at the end of the blog?